### PR TITLE
Implement likes

### PR DESCRIPTION
**Pull request by jk1dd**

### CODE REVIEW

**Comment by jk1dd** [comment]
still need to do some refactoring, but functionality in place

**Comment by jk1dd** [comment]
@case-eee this is the PR with the most recent implementation of likes - are we supposed to merge this in if everything is looking good, or wait for individual review?

File: spec/models/like_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe Like do
```
Comment (case-eee) [inline spec/models/like_spec.rb:3]: i'd like to see unit tests for the relationships on `Comment` and `Binary`

File: spec/models/comment_spec.rb [hunk 2]
```diff
@@ -44,6 +44,13 @@
       expect(user.home.binaries.first.comments.first).to eq(comment)
       expect(user.home.binaries.first.comments.first).to be_a(Comment)
     end
+  end
+
+  context 'factory' do
+    it 'creates a valid comment' do
```
Comment (case-eee) [inline spec/models/comment_spec.rb:50]: what about an invalid comment? do we have a test for that?

File: app/models/like.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
```
Comment (case-eee) [inline app/models/like.rb:1]: can a user like the same resource more than one time?

File: app/models/like.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+  belongs_to :user
+  belongs_to :likeable, polymorphic: true
```
Comment (case-eee) [inline app/models/like.rb:3]: can a "Like" be created without the "likable" content or the "user"? we should probably have validations on these foreign_key fields 👍

**Review by case-eee — COMMENTED** [review]
@jk1dd left you some comments. i'd like to see more tests for the functionality that you added. overall, great job 👍

**Pull request merged**